daemon/src/oracle.rs

```diff
@@ -362,11 +362,13 @@ impl xtra::Actor for Actor {
                             .await;
                     }
                 }
-                this.clone()
-                    .send_interval(SYNC_ATTESTATIONS_INTERVAL, || SyncAttestations)
-                    .await;
             }
         });
+
+        self.tasks.add(async {
+            this.send_interval(SYNC_ATTESTATIONS_INTERVAL, || SyncAttestations)
+                .await;
+        })
```

Comment:
This was on purpose in the same task to only start it after we fetched everything from the db.

Reply:
Ooops, my bad. I thought there was no point and it made rebasing easier. I'll undo it!